backend/src/modules/tournament-tables/core/domain/TableNumber.ts

```diff
@@ -0,0 +1,12 @@
+export class TableNumber {
+  private readonly TYPE = 'TableNumber';
+
+  private constructor(readonly raw: number) {}
+
+  static from(tableNumber: number): TableNumber {
+    if (tableNumber <= 0 || tableNumber > 200) {
```

Comment:
jeśli się nie mylę to warunek przepuści nam teraz także liczby większe niż 200 :P

Reply:
warto by było napisać do tego testy :P tak sobie teraz pomyślałem

Reply:
No ten warunek nie jest raczej dobry i nie ma dla niego testu. Możesz dodać test w domain dla tworzenia TableNumber.

Reply:
dobra, już widzę, mój błąd :P wybaczcie, nie w tę stronę 'sprawdzałem w głowie"